tests/is/L42/tests/L42Source/TestJavaServer/This.L42

```diff
@@ -17,7 +17,7 @@ LoadDeploy = Load:{reuse[L42.is/Deploy]}
 DeployGit = LoadDeploy.with(writer=GW)
 TaskDeployJavaServer = DeployGit.module(Lib()
   name='Top
-  on=Url"github.com/Language42/is/JavaServer.L42"
+  on=Secret.repo(),
```

Comment:
Ok, so in this way the various 'secret' files are now different from each others?
how is your script handling that?